examples/caption_generation.ipynb

```diff
@@ -0,0 +1,1277 @@
+{
```

Comment:
<p>You can copy paste the entire section from this recent notebook - https://colab.research.google.com/github/visual-layer/fastdup/blob/main/examples/analyzing-roboflow-datasets.ipynb</p><p> </p><p>I've yet to update all notebooks using the updated sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/visual-layer/fastdup/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>